backend/Tasque.WebAPI/Program.cs

```diff
@@ -24,13 +25,24 @@
 builder.Services.AddCors();
 
 builder.Services.AddDbContext<DataContext>(
-    o => o.UseNpgsql(builder.Configuration.GetConnectionString("TasqueDb"), 
+    o => o.UseNpgsql(builder.Configuration["ConnectionStrings:TasqueDb"], 
         b => b.MigrationsAssembly(typeof(DataContext).Assembly.FullName))
         .EnableDetailedErrors());
 
+AppConfigurationExtension.RegisterServices(builder.Services, builder.Configuration);
+
+builder.Services.RegisterIdentity();
 
 var app = builder.Build();
 
+using (var scope = app.Services.CreateScope())
+{
+    var dbContext = scope.ServiceProvider.GetRequiredService<DataContext>();
+    AppContext.SetSwitch("Npgsql.EnableLegacyTimestampBehavior", true);
```

Comment:
Do we really need this?

Reply:
We do not, it has accidentally leaked in the PR